frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/configs/UICustomizations.js

```diff
@@ -635,17 +635,17 @@ export const UICustomizations = {
 
       return {
         url: "/egov-workflow-v2/egov-wf/businessservice/_search",
-        params: { tenantId, businessServices:"MR" },
+        params: { tenantId, businessServices: Digit?.Customizations?.["commonUiConfig"]?.getBusinessService("muster roll") },
         body: {
          
         },
         config: {
           enabled: true,
           select: (data) => {
-            const states =  data?.BusinessServices?.[0]?.states?.filter(state=> state.applicationStatus)?.map(state=> {
+            const states =  data?.BusinessServices?.[0]?.states?.filter(state=> state.state)?.map(state=> {
               return {
-                "i18nKey":`WF_MUSTOR_${state?.applicationStatus}`,
-                "wfStatus":state?.applicationStatus
+                "i18nKey":`WF_MUSTOR_${state?.state}`,
```

Comment:
@riyamathew-eGov localise this i18nKey properly WF_<business_name>_STATUS_<status_name>

Reply:
also call get transformed local function on the locale code it it has dots in it like EXPENSE.WAGES